File: MirWebAPI/Models/Map.cs
using System;
using System.Collections.Generic;

namespace MirWebAPI.Models
{
    public class Map
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string NameKR { get; set; }
        public string Description { get; set; }
        public DateTime LastUpdate { get; set; }

        public ICollection<Monster> Monsters { get; set; }
        public ICollection<Npc> Npcs { get; set; }
        public ICollection<Floor> Floors { get; set; }
    }
}

File: MirWebAPI/src/app/_models/map.ts
import { Monster } from "./monster";
import { Floor } from "./floor";
import { Npc } from "./npc";

export interface Map {  
    id: number;
    name: string;
    nameKR: string;
    description: string;
    lastUpdate: Date;
    monsters?: Monster[];
    floors: Floor[];
    npcs?: Npc[];
}
